Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -158,13 +170,42 @@ - (void)loadForDisplayWithTransaction:(YapDatabaseReadTransaction *)transaction
                                                       relation:NSLayoutRelationGreaterThanOrEqual],
         ]];
     } else {
-        [self.viewConstraints addObjectsFromArray:@[
-            [self.messageBubbleView autoPinEdgeToSuperviewEdge:ALEdgeLeading
-                                                     withInset:self.conversationStyle.gutterLeading
-                                                      relation:NSLayoutRelationGreaterThanOrEqual],
-            [self.messageBubbleView autoPinEdgeToSuperviewEdge:ALEdgeTrailing
-                                                     withInset:self.conversationStyle.gutterTrailing],
-        ]];
+        if (self.shouldHavesendFailureBadge) {
+            self.sendFailureBadgeView = [UIImageView new];
+            self.sendFailureBadgeView.image =
+                [self.sendFailureBadge imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
+            self.sendFailureBadgeView.tintColor = [UIColor ows_destructiveRedColor];
```

Comment:
No change needed now, but WRT earlier conversations, maybe we should default to more "component" named colors as opposed to spectrum, e.g. "failureBadgeColor" vs. "redColor"

Reply:
We only use this color once in this class, so I wasn't going to make a method - hence there's no local name for this color.  I like that `ows_destructiveRedColor` alludes to the fact that this is used for "errors & destructive actions" - although we could improve it.  I don't mind that the color "red" is in the name, since this color is the same in all themes.